src/components/MenuItem.tsx

```diff
@@ -477,8 +484,12 @@ function MenuItem(
             titleToWrap = html;
         }
 
+        if (shouldTruncateTitle) {
+            titleToWrap = Parser.truncateHTML(titleToWrap, limit, {ellipsis: '...'});
```

Comment:
I think this one also needs to be updated. 
```suggestion
            titleToWrap = Parser.truncateHTML(titleToWrap, characterLimit, {ellipsis: '...'});
```